applications/ShapeOptimizationApplication/tests/algorithm_penalized_projection_test/parameters.json

```diff
@@ -63,10 +63,19 @@
         },
         "output" : {
             "design_output_mode" : "WriteDesignSurface",
-            "nodal_results"      : [ "SHAPE_CHANGE" ],
+            "nodal_results"      : [
```

Comment:
Is there a reason why you added more output? I thought the less the better in case of the tests.

Reply:
The less the better is now no output (by default). That would allow us to have the full output that makes sense for this case prepared and then simply switch it on when needed.

Reply:
however that is a different discussion and i revert it here